t/config.yml

```diff
@@ -0,0 +1,13 @@
+multi_option:
```

Comment:
not very important, but it would be neat of you mock the `load_files` sub instead of adding this file

Reply:
hmm I think its ok for reading file. the Multi option here is purely to test this functionality added in code even though we don't use it for anything yet.

Reply:
understood, but you'd find people argue that you should isolate your tests even from filesystem access, as I said not very important.